### ISSUE TITLE
ValueError: Keyword RTEMP is not properly terminated

### ISSUE THREAD

**Issue by tklov11** [equinor/ecl2df#210]
Hi,

I try to access the compdat content using EclFiles as shown below:

f = ecl2df.Eclfiles('GINAKROG_PRED0_B5_WELL3_NEWPARAM.DATA')
c = ecl2df.compdat.df(f)

This gives the following error:

  File "<stdin>", line 1, in <module>
  File "/prog/res/komodo/2020.09.02-py36/root/lib/python3.6/site-packages/ecl2df/compdat.py", line 305, in df
    compdat_df = deck2dfs(eclfiles.get_ecldeck())["COMPDAT"]
  File "/prog/res/komodo/2020.09.02-py36/root/lib/python3.6/site-packages/ecl2df/eclfiles.py", line 97, in get_ecldeck
    deck = opm.io.Parser().parse(deckfile, parsecontext)
ValueError: Keyword RTEMP is not properly terminated

However, RTEMP looks properly terminated to me in the .DATA file. This error occurs in both the github and Komodo versions. The model runs fine in Eclipse.

Link:
.DATA file: /scratch/gina_krog/mpic/AICD_B5/eclipse-0/model/GINAKROG_PRED0_B5_WELL3_NEWPARAM.DATA

**Comment by berland** [comment]
Thanks. Can you split into two issues?

**Comment by tklov11** [comment]
Awh, do I really have to?

**Comment by berland** [comment]
Do you foresee that one fix ( = one PR) will fix both problems? If not they should be split, so that each of then can be discussed and resolve/closed  separately. 

Just make a new issue for the second where you copy in the text from this issue, and then edit the this issue.

**Comment by tklov11** [comment]
Ok then :-)

**Comment by tklov11** [comment]
Does this look better?

**Comment by berland** [comment]
I have edited the title, as the reference to Gina Krog does not matter for the bug.

Can you share details to make this reproducible without access to Gina Krog? Can you add RTEMP to the Reek example case in ecl2df f.ex and trigger the same?

You can copy the RTEMP statement into this issue.

**Comment by tklov11** [comment]
Shure I can. I will start to reproduce RTEMP in Reek. I have to think about how to share details on the other issue, see if I can reproduce this also in Reek.

**Comment by tklov11** [comment]
I included RTEMP and some arbitrary reservoir temperature in the reek model, see attached file. This is what I got.

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/prog/res/komodo/2020.11.rc6-py36-rhel7/root/lib/python3.6/site-packages/ecl2df/compdat.py", line 425, in df
    compdat_df = deck2dfs(eclfiles.get_ecldeck())["COMPDAT"]
  File "/prog/res/komodo/2020.11.rc6-py36-rhel7/root/lib/python3.6/site-packages/ecl2df/eclfiles.py", line 98, in get_ecldeck
    deck = opm.io.Parser().parse(deckfile, parsecontext)
ValueError: 
Failed to parse keyword: RTEMP
In file /private/tklov/mygit/ecl2df/tests/data/reek/eclipse/model/2_R001_REEK-0.DATA, line 208

Error message: Malformed floating point number 'REGIONS'



[2_R001_REEK-0.LOG](https://github.com/equinor/ecl2df/files/5607864/2_R001_REEK-0.LOG)

**Comment by berland** [comment]
@asny, is this something you have fixed in https://github.com/OPM/opm-common/commit/74c3bf82fbbaa3ee8aeb4aedfff7ab6f03634324 ?

**Comment by asny** [comment]
I guess you meant @asnyv

**Comment by berland** [comment]
Yes I did, sorry the disturbance..

**Comment by berland** [comment]
https://github.com/OPM/opm-common/issues/519 looks indeed like the same problem.

**Comment by asnyv** [comment]
Yes, it is the same issue @berland and @tklov11. Ran into it through `ecl2df` as well. 

And unfortunately asny was taken when I made my account 😅

**Comment by berland** [comment]
The fix (merged Sep 28) did not get into the released OPM 2020.10, have to wait until OPM 2021.04

**Comment by asnyv** [comment]
@berland that's disappointing 😕 Merge was several weeks before the feature freeze for 2020.10, so just assumed it was going to be there 🤔

Just in case: do we know for sure that it actually is fixed? 😅 

As mentioned in the PR in opm-
common, it was never actually tested that the change PR fixed the issue. Just seemed _very_ likely, and got a supporting comment from a maintainer. 

If we are unsure, it might be worth spending a bit of time to build opm-common from source to make sure that we at least get it in 6 months...

**Comment by asnyv** [comment]
@berland Sure you are using the most recent version of `opm` from PyPi? 

Seems to me like the fix is in place 🤔 I just tested with `opm==2020.10.2`, and the issues I had with `2020.10.1` are gone. Could it be that there is a mixup since `2020.10.1` annoyingly was released 2020-06-25? 

Also tested that the issue is still there with `opm==2020.10.1`, to make sure the fix wasn't caused by a data input change.
